app/javascript/player/index.jsx

```diff
@@ -57,6 +57,13 @@ class Player extends React.Component<Props, { playing: boolean }> {
     this.audioElement = document.createElement('audio')
   }
 
+  componentDidMount () {
+    const Url = new URL(window.location.href)
+    if (Url.searchParams.get('autoplay') === 'true') {
```

Comment:
Presence is a more standard way to do a boolean with get params. Let’s send people to `/?autoplay` and check with `url.searchParams.has('autoplay')`